apache-couchbase-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,83 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'apache-couchbase',
+        rules: [
+          {
+            alert: 'ApacheCouchbaseHighCPUUsage',
```

Comment:
wasn't this renamed just couchbase in this PR?
https://github.com/grafana/jsonnet-libs/pull/1034

Reply:
@v-zhuravlev It seems like I accidentally restored the old /apache-couchbase-mixin dir, but all the corrections are made in the /couchbase-mixin directory. I just pushed changes to remove the old directory.